src/Control/Monad/Trans/Tardis.hs

```diff
@@ -53,6 +54,8 @@ import Control.Monad.Morph
 -- a 'backwards-traveling' state and a 'forwards-traveling' state.
 -- This library consistently puts the backwards-traveling state first
 -- whenever the two are seen together.
+--
+-- Note that even if this library follows the definition of a monad transformer, as described in 'MonadTrans', this is not a true monad transformer, because the underlying 'Monad' needs to be an instance of 'MonadFix', which is not general enough for 'MonadTrans'.
```

Comment:
What definition are you referring to? "A monad transformer makes a new monad out of an existing monad" reads to me like `forall m. Monad m => Monad (t m)` which is what `TardisT` doesn't satisfy.